cmd/version.go

```diff
@@ -24,3 +24,17 @@ func MockVersion(version string) (restore func()) {
 	Version = version
 	return func() { Version = old }
 }
+
+// PersonalityInfo contains information about the application identity.
+// These values will be used across the CLI.
+type PersonalityInfo struct {
```

Comment:
After reviewing all of the PR, there's apparently no benefit in using this abstraction. How about we simply have cmd.ProgramName and cmd.DisplayName?